torch/_torch_docs.py

```diff
@@ -3152,7 +3152,16 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+Returns tensor with the same data and number of elements as :attr:`input`,
```

Comment:
Too much of our documentation starts with "returns tensor." Start with the highest order bit: 

"Flattens zero or more consecutive dimensions of :attr:\`input\`, starting from :attr:\`start_dim\` at left and proceeding until :attr:\`end_dim\` at right."